packages/portal/frontend/src/app/views/GradingPage.ts

```diff
@@ -23,6 +23,20 @@ const ERROR_MALFORMED_PAGE: string = "malformed page with info elements";
 const WARN_EMPTY_FIELD: string = "empty field";
 
 export class GradingPageView extends AdminPage {
+
+    public static UBC_LETTER_GRADES: Map<string, {lower: number, upper: number}> = new Map<string, {lower: number, upper: number}>([
```

Comment:
This will need to be checked over after merging. Was there a toggle somewhere that swapped letter grades with numerical grades?